docs/design/i18n-zh-assessment.md

```
@@ -0,0 +1,812 @@
+
+```python
+ISSUE_PATTERN = r"#\d{1,5}(?![\dA-Fa-f-])(?!\.\d)"
+```
+
+The tracker-item family has no leading `\b`, so `参见#NNN。` (with digits) is
+caught with or without spaces — verified. The process-token families are
+`\b`-anchored (`\b[Tt]ask\s+#?\d+\b`, `\bEPIC\b`, `\bH\d{1,3}\b`,
+`\bAC\d\b`, and the two phrases that locate text in repository history). Python places
+`\b` only between `\w` and `\W`, and CJK characters are `\w`, so:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)